### PR TITLE
Rework the enter verification code step

### PR DESCRIPTION
**Pull request by ethax-ross**
### Trello card

[Trello-2902](https://trello.com/c/KGD3HyIm/2902-improve-content-of-verification-code-step-in-the-get-an-adviser-funnel)

### Context

This step doesn't perform all that well and we see roughly 1/5 users drop off/don't successfully enter their verification code and progress to the next step.

In an effort to tackle this we are making the copy clearer/reworking the layout of the step (this will bring it inline with the GiT website transactions).

### Changes proposed in this pull request

- Rework the enter verification code step

### Guidance to review

The Cypress tests will need to be updated when this is shipped.

### CODE REVIEW

**Comment by gemmadallmandfe** [comment]
Looks good to me

@SaimaBodiat Do you want to have a quick look at this given that you oversaw the same change to our other funnels?

**Comment by SaimaBodiat** [comment]
looks good to me

**Review by gemmadallmandfe — APPROVED** [review]

File: app/views/wizard/steps/_authenticate.html.erb [hunk 1]
```diff
@@ -1,9 +1,18 @@
 <%
-  link = link_to("resend verification", resend_verification_path)
-  hint_text = "Check your junk mail folder or #{link}."
-  hint_text += "<br /><b>We've sent you another email.</b>" if params[:verification_resent]
+  hint_text = t("helpers.hint.wizard_steps_authenticate.timed_one_time_password.text")
+  hint_text += "<br><b>#{t('helpers.hint.wizard_steps_authenticate.timed_one_time_password.resent')}</b>" if params[:verification_resent]
```
Comment (peteryates) [inline app/views/wizard/steps/_authenticate.html.erb:3]: Think this warrants a helper method?

**Review by peteryates — APPROVED** [review]
Looks good, left a minor comment

**Review by peteryates — APPROVED** [review]

**Pull request merged**